src/DataMapper/Product/Provider/ProductIndexedPricesProvider.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusMeilisearchPlugin\DataMapper\Product\Provider;
+
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+
+final class ProductIndexedPricesProvider implements ProductIndexedPricesProviderInterface
```

Comment:
I don't like this name, what's the reasoning? Could it just be `ProductVariantPricesProvider` or `ProductPricesProvider`?